aws/sqs/process

```diff
@@ -58,6 +58,7 @@ PR="$PR" REF="$HEAD_SHA" HTML_URL="$HTML_URL" ./report > "docs/$HEAD_SHA.md"
 cat "docs/$HEAD_SHA.md"
 ./aws/s3/generate-and-upload-site
 rm "docs/$HEAD_SHA.md"
+rm -rf "docs/$HEAD_SHA"
```

Comment:
Let's add a comment for each of these lines, so we remember that `60` removes the report and `61` removes all the flamegraphs.
:+1: